pkg/webhooks/mutation.go

```diff
@@ -65,8 +65,9 @@ func (ws *WebhookServer) HandleMutation(
 
 		// gather patches
 		patches = append(patches, engineResponse.GetPatches()...)
-		if len(engineResponse.GetPatches()) != 0 {
-			logger.Info("mutation rules from policy applied succesfully", "policy", policy.Name)
+		if len(patches) != 0 {
+			rules := engineResponse.GetSuccessRules()
```

Comment:
This log is printed out per policy, we should check `len(engineResponse.GetPatches()) != 0`, not joined patches.

Reply:
updated, thanks!